src/OpenApi/src/Services/IDocumentProvider.cs

```diff
@@ -20,4 +22,5 @@ internal interface IDocumentProvider
 {
     IEnumerable<string> GetDocumentNames();
     Task GenerateAsync(string documentName, TextWriter writer);
+    Task GenerateAsync(string documentName, TextWriter writer, OpenApiSpecVersion openApiSpecVersion);
```

Comment:
I'm assuming this is effectively optional and existing redeclarations of the interface are fine not including it.

Reply:
Yep, since this interface is pseudo-public, I've guarded the consuming implementation in `GetDocumentWorker` to guard against instances where it is not implemented.